homeassistant/components/broadlink/config_flow.py

```diff
@@ -111,11 +92,18 @@ async def async_step_user(self, user_input=None):
             else:
                 device.timeout = timeout
 
-                if self.source != SOURCE_REAUTH:
+                if self.source != config_entries.SOURCE_REAUTH:
                     await self.async_set_device(device)
                     self._abort_if_unique_id_configured(
                         updates={CONF_HOST: device.host[0], CONF_TIMEOUT: timeout}
                     )
+
+                    if device.type == "Unknown":
```

Comment:
Do we have a constant for this string ?

Reply:
No, neither for the other device types. Do you think it would be reasonable to add? How about we leave that for another PR?